scripts/upload_website_demos.py

```diff
@@ -95,15 +93,28 @@ def upload_demo_to_space(
 
 demos = [demo for demo in demos if demo not in DEMOS_TO_SKIP and os.path.isdir(os.path.join(GRADIO_DEMO_DIR, demo)) and  os.path.exists(os.path.join(GRADIO_DEMO_DIR, demo, "run.py"))]
 
+def version_exists(version: str):
+    """Checks if the given version of gradio has been uploaded to pypi"""
+    for _ in range(600):
+        if requests.get("https://pypi.org/pypi/gradio/json").json()["info"]["version"] == version:
+            return True
+        time.sleep(1)
+    return False
+
```

Comment:
I don't think we should do this, as it will hog the runner until the pypi publish has completed, possibly forever if the publish fails/ doesn't trigger.

Reply:
we set the timeout so it won't go forever but you're right it will hog the runner. workflow dispatch makes more sense